docs/usage.md

```diff
@@ -68,7 +100,18 @@ The widgets within the solution pack's dashboard, such as the Record Summary Car
 The "Global Visibility" playbook collection plays a pivotal role in the FortiSOAR Global Visibility Solution Pack by aggregating, centralizing, and transforming data from individual sites into a unified dataset. This dataset is harnessed by the solution pack's widgets to provide users with insightful and consolidated visualizations of alerts, incidents, tasks, and other critical metrics across multiple FortiSOAR instances. The "Global Visibility" module acts as the foundation for enhancing decision-making and security operations efficiency.
```

Comment:
The **Global Visibility** playbook collection plays a pivotal role in FortiSOAR&trade;'s Global Visibility solution pack by aggregating, centralizing, and transforming data from individual sites into a unified dataset. This dataset is harnessed by the solution pack's widgets to provide users with insightful and consolidated visualizations of alerts, incidents, tasks, and other critical metrics across multiple FortiSOAR&trade; instances. The **Global Visibility** module acts as a foundation for enhancing decision-making and security operations efficiency.